src/cosmosdb-preview/azext_cosmosdb_preview/custom.py

```diff
@@ -604,6 +218,63 @@ def cli_cosmosdb_managed_cassandra_datacenter_update(client, resource_group_name
     return client.begin_create_update(resource_group_name, cluster_name, data_center_name, data_center_resource)
 
 
+def _handle_exists_exception(http_response_error):
+    if http_response_error.status_code == 404:
+        return False
+    raise http_response_error
+
+
+def cli_cosmosdb_graph_create(client, account_name, resource_group_name, graph_name):
+    graph = GraphResourceCreateUpdateParameters(resource=GraphResource(id=graph_name))
+    return client.begin_create_update_graph(resource_group_name, account_name, graph_name, graph)
+
+
+def cli_cosmosdb_graph_exists(client, account_name, resource_group_name, graph_name):
+    try:
+        client.get_graph(resource_group_name, account_name, graph_name)
+    except HttpResponseError as ex:
+        return _handle_exists_exception(ex)
+
+    return True
+
+
+def cli_cosmosdb_service_exists(client, account_name, resource_group_name, service_name):
```

Comment:
I didn't see `cosmosdb service exists` command. Is this command missing or is this function extra?

Reply:
will remove the function